src/main/io/osd.c

```diff
@@ -153,6 +153,13 @@ static char osdGetAltitudeSymbol()
     }
 }
 
+static char osdGetBatterySymbol(int cellVoltage)
+{
+    /* Calculate current battery cell voltage as a percentage of range */
+    int voltPercent = ((cellVoltage - (batteryConfig()->vbatmincellvoltage * 10)) * 7) / ((batteryConfig()->vbatmaxcellvoltage - batteryConfig()->vbatmincellvoltage) * 10);
```

Comment:
Blinking is already done.

The only thing I noticed with scaling over the full min-max range is that it is possible to have a low voltage warning and there still be a bar in the indicator. Although as that isn't the same as the battery being empty I'll probably just go with the full range.